### ISSUE TITLE
Encode vp8 file

### ISSUE THREAD

**Issue by martingaudioso1990** [traud/asterisk-opus#20]
Hi, I'm using vp8 format to write frames into a file. It works great but I'm having issues to read this file or encode it into a playable format. Do you know an approach to do that?

**Comment by traud** [comment]
No. I inherited that code part from @seanbright and he inherited that part from @meetecho. Personally, I never used it, so I am not even sure that part works at all. However, I guess, you have to wrap the media stream into a container format like [WebM](https://en.wikipedia.org/wiki/WebM) anyway. The code does not look like, this is done for you (yet).

**Comment by nicolasminassian90** [comment]
I did that. Thank you.
